xdebug-tunnel.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Usage:
+#   To open a tunnel: ./xdebug-tunnel.sh example_com_prod
```

Comment:
Think it's better to be explicit here with the `open` command?

```bash
./xdebug-tunnel.sh example_com_prod open
./xdebug-tunnel.sh example_com_prod close
```

Is this better?

```bash
./xdebug-tunnel.sh open example_com_prod
./xdebug-tunnel.sh close example_com_prod
```

Reads better at least